sdk/servicebus/service-bus/src/receivers/receiver.ts

```diff
@@ -38,7 +38,15 @@ export interface Receiver<ReceivedMessageT> {
    * @param handlers A handler that gets called for messages and errors.
    * @param options Options for subscribe.
```

Comment:
Nit: This TSDoc should probably include a `@returns` tag.

Reply:
Done